src/skmultiflow/transform/minmax_scaler.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+
+from skmultiflow.transform.base_transform import StreamTransform
+from skmultiflow.utils import FastBuffer, get_dimensions
+
+
+class MinmaxScaler(StreamTransform):
+    """ Transform features by scaling each feature to a given range.
+    This estimator scales and translates each feature individually such
+    that it is in the given range on the training set, e.g. between zero and one.
+    For the training set we consider a window of a given length.
+
+    Parameters
+    ----------
+    window_size: int (Default: 200)
+        Defines the window size to compute min and max values.
+
+    Examples
+    --------
+    """
+
+    def __init__(self, window_size=200):
+        super().__init__()
+        self.window_size = window_size
+        self.window = None
+
+        self.__configure()
+
+    def __configure(self):
+        self.window = FastBuffer(max_size=self.window_size)
+
+    def transform(self, X):
+        """ transform
+
+        Does the transformation process in the samples in X.
+
+        Parameters
+        ----------
+        X: numpy.ndarray of shape (n_samples, n_features)
+            The sample or set of samples that should be transformed.
+
+        """
+        r, c = get_dimensions(X)
+        for i in range(r):
+            row = np.copy([X[i][:]])
+            for j in range(c):
+                value = X[i][j]
+                min = self._get_min(j)
+                max = self._get_max(j)
```

Comment:
`max` is a reserved name in Python. Consider `max_val` and `min_val` for the corresponding variables.